src/services/arweave/SimpleArweaveDataProvider.ts

```diff
@@ -141,4 +145,38 @@ export class SimpleArweaveDataProvider implements ArweaveDataProvider {
       });
     }
   }
+  async validateArweaveId(id: string): Promise<ValidationObject> {
+    const validatedIdObject = { name: '', status: false, error: '' };
```

Comment:
i didn't intend for this type to be used here, only in the validtionInput component. 

these should throw an error, or return the transaction id